core/src/test/java/org/infinispan/conflict/impl/TxConflictResolutionTest.java

```diff
@@ -0,0 +1,98 @@
+package org.infinispan.conflict.impl;
+
+import static org.infinispan.configuration.cache.CacheMode.DIST_SYNC;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.conflict.MergePolicy;
+import org.infinispan.distribution.MagicKey;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.transaction.LockingMode;
+import org.infinispan.transaction.TransactionMode;
+import org.testng.annotations.Test;
+
+/**
+ * <p>
+ * 1. do a split, and let k -> A, k -> null in the two partitions
+ * 2. merge partitions
+ * 3. check that all members read A
+ * </p>
+ *
+ * <p>See ISPN-12725.</p>
+ *
+ * @author Ryan Emerson
+ * @since 9.1
+ */
```

Comment:
I think this needs updating or removed :slightly_smiling_face:

Reply:
Updated